api/v1beta1/provider_types.go

```diff
@@ -42,6 +42,10 @@ type ProviderSpec struct {
 	// +optional
 	Address string `json:"address,omitempty"`
 
+	// HTTP(S) address of the proxy
```

Comment:
```suggestion
	// HTTP/S address of the proxy
```

(+ a validation rule to only accept `http://` and `https://`)